backend/app/processing/db_storage.py

```
@@ -0,0 +1,57 @@
+class DatabaseStorageManager:
+    """Manages storage of database files and configurations."""
+    def __init__(self, storage_base: str = "db_storage"):
+        self.storage_base = Path(storage_base)
+        self.storage_base.mkdir(parents=True, exist_ok=True)
+
+    def get_chatbot_dir(self, chatbot_id: str) -> Path:
+        bot_dir = self.storage_base / chatbot_id
+        bot_dir.mkdir(parents=True, exist_ok=True)
+        return bot_dir
+
+    def clean_old_files(self, chatbot_id: str, keep_filenames: List[str]) -> List[str]:
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)